### PR TITLE
Changes NT relation into economic status (for real)

### PR DESCRIPTION
**Pull request by carpevenenum**
Inspired by #8587

Does what the title says, also removes the NTCC report from the start of the round because: 1) I can; 2) Nobody reads it; 3) It pretty much hinges on NT relation being a thing; 4) It isn't needed with the SCC arc; 5) It never took contractors into account.

### CODE REVIEW

**Comment by carpevenenum** [comment]
!review

File: code/__defines/mobs.dm [hunk 1]
```diff
@@ -226,13 +226,13 @@
 #define IPC_OWNERSHIP_PRIVATE "Privately Owned"
 
 // NT's alignment towards the character
```
Comment (carpevenenum) [inline code/__defines/mobs.dm:228]: note to self: change this

File: code/game/antagonist/station/traitor.dm [hunk 2]
```diff
@@ -101,9 +101,9 @@ var/datum/antagonist/traitor/traitors
 	traitor_mob.faction = "syndicate"
 	spawn_uplink(traitor_mob)
 	// Tell them about people they might want to contact.
-	var/mob/living/carbon/human/M = get_nt_opposed()
+	var/mob/living/carbon/human/M = get_poor()
```
Comment (NonQueueingMatt) [inline code/game/antagonist/station/traitor.dm:104]: get_poor() is not an appropriate name for a function here. Should be `get_economic_status`.

File: code/game/jobs/job/job.dm [hunk 3]
```diff
@@ -79,22 +79,22 @@
...
 	if(H.client)
-		switch(H.client.prefs.nanotrasen_relation)
-			if(COMPANY_LOYAL)        loyalty = 1.30
-			if(COMPANY_SUPPORTATIVE) loyalty = 1.15
-			if(COMPANY_NEUTRAL)      loyalty = 1
-			if(COMPANY_SKEPTICAL)    loyalty = 0.85
-			if(COMPANY_OPPOSED)      loyalty = 0.70
+		switch(H.client.prefs.economic_status)
+			if(ECONOMICALLY_WEALTHY)        econ_status = 1.50
+			if(ECONOMICALLY_WELLOFF) 		econ_status = 1.25
```
Comment (NonQueueingMatt) [inline code/game/jobs/job/job.dm:86]: The higher levels should be kept as is imho. 1.50 is too big of a bonus.

File: code/modules/client/preferences.dm [hunk 4]
```diff
@@ -140,7 +140,7 @@ datum/preferences
 	var/list/ccia_actions = list()
 	var/list/disabilities = list()
 
-	var/nanotrasen_relation = "Neutral"
+	var/economic_status = "Average"
```
Comment (NonQueueingMatt) [inline code/modules/client/preferences.dm:143]: Use the define.

File: code/modules/client/preferences.dm [hunk 5]
```diff
@@ -656,7 +656,7 @@ datum/preferences
 		ccia_actions = list()
 		disabilities = list()
 
-		nanotrasen_relation = "Neutral"
+		economic_status = "Average"
```
Comment (NonQueueingMatt) [inline code/modules/client/preferences.dm:659]: Ditto, define.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/antagonist/station/traitor.dm (hunk 2 above)
Comment (carpevenenum) [reply code/game/antagonist/station/traitor.dm:104]: Disagree, this proc is only used to get people of low status.

File: code/game/jobs/job/job.dm (hunk 3 above)
Comment (carpevenenum) [reply code/game/jobs/job/job.dm]: Resolved, however statuses below average are still more poor than they used to be, though not so drastically.

File: code/modules/client/preferences.dm (hunk 4 above)
Comment (carpevenenum) [reply code/modules/client/preferences.dm]: Resolved

File: code/modules/client/preferences.dm (hunk 5 above)
Comment (carpevenenum) [reply code/modules/client/preferences.dm]: Resolved

File: code/game/antagonist/station/traitor.dm (hunk 2 above)
Comment (NonQueueingMatt) [reply code/game/antagonist/station/traitor.dm:104]: True.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — CHANGES_REQUESTED** [review]
migration's out of date

**Comment by NonQueueingMatt** [comment]
Dead author.

**Pull request closed without merging**